examples/with-typescript-mantine/pages/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
This file does not seem to showcase Mantine, I would remove it to keep the example focused on its goal.